docs/dangerfile.js

```diff
@@ -1,5 +1,5 @@
 import { warn } from 'danger';
-import fs from 'fs';
+const fs = require('fs');
```

Comment:
Why? 🤔

Reply:
I saw in Cedric's PR that there was a dangerjs error with `import` here. It might not actually be an issue, so I am reverting this line back to what is one main (`import fs from ...`)

Reply:
<img width="918" alt="Screen Shot 2022-07-13 at 5 07 00 PM" src="https://user-images.githubusercontent.com/6455018/178835179-0f629309-a622-4ff3-9e60-8ebc2103c7b0.png">

 weird right?

Reply:
oh my god, looks like it uses https://www.npmjs.com/package/require-from-string to require this file 😱